core/templates/dev/head/services/SearchServiceSpec.ts

```diff
@@ -312,25 +272,25 @@ describe('Search service', function() {
       };
       var urlComponent = '?q=protractor%20test&category=("Mathematics")' +
                          '&language_code="en" OR "ar")';
-      expect(function() {
-        SearchService.updateSearchFieldsBasedOnUrlQuery(urlComponent, results);
-      }).toThrow(new Error('Invalid search query url fragment for ' +
-                           'languageCodes: language_code="en" OR "ar")'));
+      SearchService.updateSearchFieldsBasedOnUrlQuery(urlComponent, results);
       expect(results.languageCodes.selections).toEqual({});
       expect(results.categories.selections).toEqual({
         Mathematics: true
       });
 
       var urlComponent = '?q=protractor%20test&category=("Mathematics")' +
                          '&language_code="en" OR "ar"';
-      expect(function() {
-        SearchService.updateSearchFieldsBasedOnUrlQuery(urlComponent, results);
-      }).toThrow(new Error('Invalid search query url fragment for ' +
-                           'languageCodes: language_code="en" OR "ar"'));
+      SearchService.updateSearchFieldsBasedOnUrlQuery(urlComponent, results);
       expect(results.languageCodes.selections).toEqual({});
       expect(results.categories.selections).toEqual({
         Mathematics: true
       });
+
+      var urlComponent = '?q=protractor%20test&category="Mathematics"' +
```

Comment:
When a line is too long, use parens to bound it and break after the left paren:

```
    var urlComponent = (
      '?q=protractor%20test&category="Mathematics"&language_code="en" OR "ar"');
```

Also make sure to indent the following line by 2 spaces.

Reply:
Done.